raindrops/olegbet/raindrops.go

```diff
@@ -0,0 +1,15 @@
+package raindrops
+
+import "strconv"
+
+func Convert(number int) string {
+	var result string
+	if number % 3 == 0 {result += "Pling"}
+	if number % 5 == 0 {result += "Plang"}
+	if number % 7 == 0 {result += "Plong"}
+	if result == "" {
+		return strconv.Itoa(number)
```

Comment:
```suggestion
		result += strconv.Itoa(number)
```
This way you can remove the else clause and just add `return` at the end (with named returns).